ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx

```diff
@@ -190,6 +270,43 @@ function VulnReportsPage() {
                     </Card>
                 </PageSection>
             </PageSection>
+            {reportIdToDelete !== '' && (
+                <Modal
+                    variant="small"
+                    title="Permanently delete report?"
+                    isOpen={isModalOpen}
+                    onClose={closeDeleteModal}
+                    actions={[
+                        <Button
+                            key="confirm"
+                            variant="danger"
+                            isLoading={isDeleting}
```

Comment:
What do you think about `isDisabled={isDeleting}` in addition to `isLoading` prop?

Otherwise, someone could click again while delete is in progress?

Reply:
Good point. I assumed that the PatternFly button might have automatically prevented such a thing, but it makes sense to explicitly disable it too